x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -971,6 +971,81 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         expect(lastBreadcrumb).to.be(theCase.title);
       });
     });
+
+    describe('reporter', () => {
+      createOneCaseBeforeDeleteAllAfter(getPageObject, getService);
+
+      it('should render the reporter correctly', async () => {
+        const reporter = await cases.singleCase.getReporter();
+
+        const reporterText = await reporter.getVisibleText();
+
+        expect(reporterText).to.be('elastic');
+      });
+    });
+
+    describe('participants', () => {
+      before(async () => {
+        await createUsersAndRoles(getService, users, roles);
+        await cases.api.activateUserProfiles([casesAllUser, casesAllUser2]);
+      });
+
+      afterEach(async () => {
+        await cases.api.deleteAllCases();
+      });
+
+      after(async () => {
+        await cases.api.deleteAllCases();
+        await deleteUsersAndRoles(getService, users, roles);
+      });
+
+      it('should render the participants correctly', async () => {
+        const theCase = await createAndNavigateToCase(getPageObject, getService);
+        await cases.api.createAttachment({
+          caseId: theCase.id,
+          params: {
+            type: AttachmentType.user,
+            comment: 'test',
+            owner: 'cases',
+          },
+          auth: { user: casesAllUser, space: 'default' },
+        });
+
+        await cases.singleCase.refresh();
+        await header.waitUntilLoadingHasFinished();
+
+        const participants = await cases.singleCase.getParticipants();
+
+        const casesAllUserText = await participants[0].getVisibleText();
+        const elasticUserText = await participants[1].getVisibleText();
+
+        expect(casesAllUserText).to.be('cases all_user');
+        expect(elasticUserText).to.be('elastic');
+      });
+
+      it('should render assignees in the participants section', async () => {
+        await createAndNavigateToCase(getPageObject, getService);
+
+        await cases.singleCase.openAssigneesPopover();
+        await cases.common.setSearchTextInAssigneesPopover('case');
+        await cases.common.selectFirstRowInAssigneesPopover();
+        await cases.singleCase.closeAssigneesPopover();
+        await header.waitUntilLoadingHasFinished();
+        await testSubjects.existOrFail('user-profile-assigned-user-cases_all_user-remove-group');
+
+        const participants = await cases.singleCase.getParticipants();
+
+        expect(participants.length).to.be(3);
+
+        const casesAllUserText = await participants[0].getVisibleText();
+        const elasticUserText = await participants[1].getVisibleText();
+        const testUserText = await participants[2].getVisibleText();
+
+        expect(casesAllUserText).to.be('cases all_user');
```

Comment:
Which of them is the assignee?

Reply:
The `casesAllUserText`. I will put a comment.